packages/create-sitecore-jss/src/templates/nextjs/src/lib/layout-service-factory.ts

```diff
@@ -26,6 +26,7 @@ export class LayoutServiceFactory {
             For this, specify the number of retries the GraphQL client will attempt. 
             It will only try the request once by default.
             retries: 'number' 
+            Additionally, you can customize the retry strategy by providing your own RetryStrategy object.
```

Comment:
How should users pass custom retryStrategy?
Graphql layout and dictionary service contsructors do not accept a retry strategy param.